src/applications/timeouts.js

```diff
@@ -1,3 +1,5 @@
+import { assign } from 'src/utils/assign.js';
```

Comment:
i made a util for this because babel-plugin-transform-object-rest-spread hasn't been upgraded to work with babel 7 yet (https://github.com/babel/babel/issues/6084) without a bunch of warnings and my OCD kicked in when I saw the warnings so I just made this instead.